ui_framework/dist/ui_framework.css

```diff
@@ -706,6 +706,7 @@ body {
   /* 2 */
   font-size: 10px !important;
   /* 2 */
+  -webkit-transition: background-color 0.1s linear;
```

Comment:
Not using yarn but I was on 5.2.11. I installed fresh, now I have 5.2.17 and I'm generating the same css as you (with the lines removed).